Account & Project/2. Concepts/1. 使用者角色與權限角色總覽/6. 其他操作.md

```diff
@@ -1 +1,31 @@
+---
+title: role-other | en
+tags: Guide, Service, EN
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+<style>
+.fa-times{color:#ADADAD; font-size:25px}
+.fa-check{color:#27a5bd; font-size:25px}
+</style>
+
+
+# Comparison of role permissions of TWCC： Other Operations
+
+
+| Function | Tenant Admin | Tenant User |
+| -------- | -------- | -------- |
+|View personal API key|<i class="fa fa-check" aria-hidden="true"></i>|<i class="fa fa-check" aria-hidden="true"></i>|
+|View API key of project members|<i class="fa fa-times" aria-hidden="true"></i>|<i class="fa fa-times" aria-hidden="true"></i>|
```

Comment:
[SUG]

View API key (of project members)*3 > View all (API keys in the project)